data/fixtures/scopes/textual/collectionItem.textual.iteration6.scope

```diff
@@ -0,0 +1,49 @@
+[
+    1,
+    2,
+    3,
+]
+---
+
+[#1 Range] = 1:4-3:6
+      >--
```

Comment:
`>--` Doesn't include the trailing whitespace before the "1" item, since we don't consider that to be part of the list scope, but the whitespace before "3" is; that's why these look unbalanced.

If there was trailing whitespace after the "3," item that would similarly not be part of the iteration range, and not included in closing marker.

Andreas says this is the same logic that handles the "inside" code, we haven't seen it in this review.

Reply:
Andreas says: if you have only whitespace between delimiters and you say take inside, you get all of the whitespace; if there's any non whitespace characters it gets trimmed.